semgrep/setup.py

```diff
@@ -101,9 +101,6 @@ def find_executable(env_name, exec_name):
         "attrs>=19.3.0",
         "colorama>=0.4.3",
         "requests>=2.22.0",
-        # Pin exact version of 'ruamel.yaml' because of unstable API.
-        # Ensure default type is still 'rt' (round-trip) and that RoundTrip
```

Comment:
This part is now guaranteed by the `test_default_yaml_type_safe` unit test, so we no longer need to manually ensure this.